tests/lib/util/get-string-from-node.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * @fileoverview Utility for retrieving callee identifier node from a CallExpression
+ * @author Automattic
+ * @copyright 2016 Automattic. All rights reserved.
+ * See LICENSE.md file in root directory for full license.
+ */
+
+var assert = require( 'assert' );
+var getStringFromNode = require( '../../../lib/util/get-string-from-node.js' );
+var config = require( '../../../.eslintrc.json' );
+var parser = require( config.parser )
+
+function getTranslatableStringsFromCode( code ) {
+	var programNode = parser.parse( code, config.env );
+	// Espree thinks it's parsing a whole program, so we just need to peel away
+	// the 'Program' packaging.
+	var stringNode = programNode.body[0].expression;
```

Comment:
Looks like we should update this project to use `eslint-config-wpcalypso`. We've since changes the linting rules to warn on this usage (rather, prefer spaces in array access, even when using number or string).